sw/airborne/modules/energy/electrical.c

```diff
@@ -212,4 +214,13 @@ void electrical_periodic(void)
     }
   }
 
+  float power = electrical.vsupply * electrical.current;
+  electrical.avg_power += power;
+  electrical.avg_cnt++;
```

Comment:
with a unit16 and a 10Hz periodic, it will restart to 0 after 110 minutes. Is it really what you want ?